src/routes/signin/providers/utils.ts

```diff
@@ -255,21 +255,39 @@ export const initProvider = <T extends Strategy>(
           req: RequestWithState<ProviderCallbackQuery, { user: string }>,
           accessToken: string,
           refreshToken: string,
-          idToken: string,
+          params: unknown,
```

Comment:
We must ensure this change does not break the Apple provider.

Reply:
[There](https://github.com/nhost/hasura-auth/blob/6b3df9a8870186f6ef270383fa318132e932f4bd/src/routes/signin/providers/utils.ts#L274) the params refers to the token as a string for the Apple integration (as before), while for the new [Azure AD](https://github.com/nhost/hasura-auth/blob/6b3df9a8870186f6ef270383fa318132e932f4bd/src/routes/signin/providers/utils.ts#L283) integration the params is an object containing the token.
IMO it behaves as before but I did not test that part as we don't use Apple SSO internally.